app/src/main/java/org/mozilla/focus/navigation/MainActivityNavigation.kt

```diff
@@ -136,7 +136,9 @@ class MainActivityNavigation(
      * Show first run onboarding.
      */
     fun firstRun() {
-        val onboardingFragment = if (Features.ONBOARDING) {
+        val onboardingFragment = if (FocusNimbus.features.onboardingWithCfrs.value().isOnboardingWithCfrs ||
+            FocusNimbus.features.onboardingWithoutCfrs.value().isOnboardingWithoutCfrs
+        ) {
```

Comment:
This would indicate we need an `onboarding` feature with a `variable` of `is-enabled`.